modules/shared/hooks/useEnsResolvers.ts

```diff
@@ -0,0 +1,46 @@
+import { CHAINS } from '@lido-sdk/constants'
+import { getStaticRpcBatchProvider } from '@lido-sdk/providers'
+import { useWeb3 } from 'modules/blockChain/hooks/useWeb3'
+import { useConfig } from 'modules/config/hooks/useConfig'
+import { useCallback, useMemo } from 'react'
+
+export const useEnsResolvers = () => {
+  const { chainId } = useWeb3()
+  const { getRpcUrl } = useConfig()
+
+  const ethProvider = useMemo(() => {
+    const rpcUrl = getRpcUrl(chainId)
+    return getStaticRpcBatchProvider(chainId, rpcUrl)
+  }, [chainId, getRpcUrl])
+
+  const lookupAddress = useCallback(
+    async (address: string) => {
+      // ENS name is not supported on Holesky for our current setup
```

Comment:
There ensName hook witch works with holeky
https://github.com/lidofinance/dao-voting-ui/blob/develop/modules/shared/hooks/useEnsNames.ts

Reply:
We'll merge those in the next release